### PR TITLE
Update strings_settings.xml

### PR DESCRIPTION
**Pull request by lianergoist**
added some danish strings

### CODE REVIEW

File: app/src/main/res/values-da/strings_settings.xml [hunk 1]
```diff
@@ -5,8 +5,8 @@
     <string name="button_select_favorites">Vælg favoritter</string>
     <string name="button_sort_az">Sort A-Z</string>
```
Comment (falzonv) [inline app/src/main/res/values-da/strings_settings.xml:6]: Is this one also correct in Danish? (this is a new string)

**Comment by falzonv** [comment]
Thank you!
Regarding the string "Sort A-Z" that has been added, is it also valid in Danish or should it be translated?

**Comment by lianergoist** [comment]
No. Sorry I missed that. Direct translation would be "Sortér A-Å", but I think "Sortér alfabetisk" is better.

**Comment by falzonv** [comment]
Perfect, thank you!
Yes, I now realize when seeing the translations that "Sort A-Z" was not a very good choice, I will replace it by "Sort alphabetically" even in the English translation.

File: app/src/main/res/values-da/strings_settings.xml (hunk 1 above)
Comment (falzonv) [reply app/src/main/res/values-da/strings_settings.xml:6]:
```suggestion
    <string name="button_sort_az">Sortér alfabetisk</string>
```

**Pull request merged**